modules/acm/src/main/java/edu/gemini/epics/acm/CaObserveSenderImpl.java

```diff
@@ -803,7 +803,12 @@ else if (val.isError()) {
                 return idleState;
             }
             else {
-                return this;
+                if(val.isIdle() && observeState != idleObserveState) {
```

Comment:
If the main CAR is changing to IDLE here, and the observe CAR has already change, then assume that the main CAR change to BUSY was lost.

Reply:
is this the one condition that sometimes happens with GNIRS?

Reply:
Yes.